### PR TITLE
Installation

### LINKED ISSUE #2: Add install script to make setup easier

### PR DESCRIPTION
**Pull request by fippli**
- Adds installation script for easier setup, fixes #2 
- Updates README
- Adds the path to the source in the script for relative references in the code

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+ln -s $(pwd)/todo.sh ~/bin/todo
```
Comment (bombsimon) [inline install.sh]:
```suggestion
ln -s $(pwd)/todo.sh ${INSTALL_DIR:-"~/bin/todo"}
```

I don't have `~/bin` but I have `~/.bin` in my `PATH`

File: todo.sh [hunk 2]
```diff
@@ -11,7 +19,7 @@ function mainMenu() {
   echo "       - - - - - - - - - - - - - - - - - - - - - -    "
   readTasks
   echo "       - - - - - - - - - - - - - - - - - - - - - - "
-  read -r -e -p  "       [A]DD / [D]ELETE / [E] EDIT / [Q]UIT : " CHOICE
+  read -r -e -p  "       [A]DD / [D]ELETE / [E]DIT / [Q]UIT : " CHOICE
```
Comment (bombsimon) [inline todo.sh:22]: Oops, nice :)

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+
+chmod +x ~/bin/todo
```
Comment (bombsimon) [inline install.sh]:
```suggestion
```

`todo` is executable so the symlink will be as well.

**Review by bombsimon — APPROVED** [review]
LGTM

**Comment by fippli** [comment]
Great! Thanks! Will make the chages :)

**Pull request merged**